linty_fresh/main.py

```diff
@@ -72,7 +72,7 @@ def create_parser() -> argparse.ArgumentParser:
         existing_problems = await storage_engine.get_existing_problems()
         problems = problems.difference(existing_problems)
 
-    awaitable_array.extend([reporter.report(problems) for
+    awaitable_array.extend([reporter.report(linter, problems) for
```

Comment:
I think we should add an option for passing this as linter reuse won't make sense otherwise. For example `passthrough` when the linter that found the issue was `stringslint`.